packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/mockMessageFrom..st

```diff
@@ -0,0 +1,23 @@
+accessing
+mockMessageFrom: aChatId
+	
+	^ TCCEvent newFromTDLibEvent: (Dictionary newFrom: {
+		'@type' -> 'updateNewMessage'.
+		'message' -> (Dictionary newFrom: {
+			'@type' -> 'message'.
+			'id' -> 9447669760.
+			'chat_id' -> aChatId.
+			'sender' -> (Dictionary newFrom: {
+				'@type' -> 'messageSenderUser'.
+				'user_id' -> 1673511834
+				}).
+			'content' -> (Dictionary newFrom: {
+				'@type' -> 'messageText'.
+				'text' -> (Dictionary newFrom: {
+					'text' -> 'Hallo'
+					})
+				}).
+				'is_outgoing' -> false.
+				'date' -> 1621267079
+			})
+		})
```

Comment:
Please take a look at TCTMocks, I believe the mocks already exist there. If they don't, putting these messages into TCTMocks would be good 😄

Reply:
they are not being sent, we forgot to remove them